### PR TITLE
Add documentation for Knative Eventing Mesh

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Add documentation

**Related issue(s)**
See also #6312

### CODE REVIEW

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+2. The HTTP Source adapter forwards the Events to the default [Knative Broker](https://knative.dev/docs/eventing/broker-trigger).
+
+3. The Knative Broker then delivers Events to the proper lambda function. 
+
+
+### Subscribe to Events 
+
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.
```
Comment (Abd4llA) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
In the new Knative Eventing mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+### Subscribe to Events 
+
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also create expressions which allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document.
```
Comment (Abd4llA) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
You can also use filter expressions which allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section in [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also create expressions which allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
+
+## Test Knative Eventing Mesh
+
+To reach the new Eventing Mesh use an HTTP request with the `/events` path.
```
Comment (Abd4llA) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
To reach the new Eventing mesh use an HTTP request with the `/events` path. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also create expressions which allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
+
+## Test Knative Eventing Mesh
+
+To reach the new Eventing Mesh use an HTTP request with the `/events` path. 
+For example, if so far you have used `gateway.example.cx/v1/events`, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh.
```
Comment (Abd4llA) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
For example, if so far you have used `gateway.example.cx/v1/events`, use `gateway.example.cx/events` to make sure you work with the new Eventing mesh. This endpoint receives only [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md) compliant Events.
```

File: docs/event-bus/01-01-event-bus.md [hunk 5]
```diff
@@ -5,3 +5,5 @@ title: Overview
 Kyma Event Bus allows you to integrate various external solutions with Kyma. To achieve successful integration, the Event Bus uses the `publish-subscribe` messaging pattern that allows Kyma to receive business Events from different solutions, enrich them, and trigger business flows using lambdas or services defined in Kyma.
 
 The Event Bus is based on the [NATS Streaming](https://github.com/nats-io/nats-streaming-server/releases) open-source log-based streaming system for cloud-native applications, which is a brokered messaging middleware and [Knative Eventing](https://knative.dev/docs/eventing/). The Event Bus provides an **at-least-once** delivery guarantee meaning the messages are retransmitted to assure they are successfully delivered at least once.
+
+>**NOTE:** You can now explore and test new functionality provided by Knative Eventing Mesh. For details, read [this](/components/event-bus/#details-knative-eventing-mesh) document.
```
Comment (kazydek) [inline docs/event-bus/01-01-event-bus.md]:
```suggestion
>**NOTE:** You can now explore and test the new functionality provided by Knative Eventing Mesh. For details, read [this](/components/event-bus/#details-knative-eventing-mesh) document.
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+---
+title: Knative Eventing Mesh (Alpha)
+type: Details
+---
+
+## Overview
+
+Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.   
+
+ >**NOTE:** Knative Eventing Mesh is available in alpha version. Use it only used for testing purposes.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
 >**NOTE:** Knative Eventing Mesh is available in the alpha version. Use it only for testing purposes.
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+---
+title: Knative Eventing Mesh (Alpha)
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
title: "Knative Eventing Mesh (Alpha)"
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+type: Details
+---
+
+## Overview
+
+Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points, and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.   
```

Comment (kazydek) [reply docs/event-bus/03-04-knative-eventing-mesh.md]: I only added a comma here: `source points, and subscribers` (it reads better this way IMHO)

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+## Overview
+
+Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.   
+
+ >**NOTE:** Knative Eventing Mesh is available in alpha version. Use it only used for testing purposes.
+ 
+ The new Eventing Mesh runs in parallel with the existing Event Bus. Sending Events to the regular eventing endpoint still uses Kyma Event Bus, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh. 
+ 
+
+
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]: too many empty lines

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+type: Details
+---
+
+## Overview
+
+Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.   
+
+ >**NOTE:** Knative Eventing Mesh is available in alpha version. Use it only used for testing purposes.
+ 
+ The new Eventing Mesh runs in parallel with the existing Event Bus. Sending Events to the regular eventing endpoint still uses Kyma Event Bus, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
 The new Eventing Mesh runs in parallel with the existing Event Bus. The Kyma Event Bus still supports sending Events to the regular eventing endpoint, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh. 
```

Comment (kazydek) [reply docs/event-bus/03-04-knative-eventing-mesh.md]: Hope I haven't changed the meaning ;)

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+ 
+ The new Eventing Mesh runs in parallel with the existing Event Bus. Sending Events to the regular eventing endpoint still uses Kyma Event Bus, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh. 
+ 
+
+
+## Event Flow
+
+### Send Events
+
+The diagram shows you the main stages of the Event flow from the moment it is sent by the external Application up to when it is received by the lambda function.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
The diagram shows you the main stages of the Event flow - from the moment the external Application sends it, up to the point when the lambda function receives it. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+ 
+
+
+## Event Flow
+
+### Send Events
+
+The diagram shows you the main stages of the Event flow from the moment it is sent by the external Application up to when it is received by the lambda function.  
+
+>**NOTE**: The flow assumes you have already used Kyma console to add a service instance of an external Application to your Namespace and created a lambda with an Event trigger.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
>**NOTE**: The flow assumes you have already added a service instance of an external Application to your Namespace in the Kyma Console and created a lambda with an Event trigger. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+### Send Events
+
+The diagram shows you the main stages of the Event flow from the moment it is sent by the external Application up to when it is received by the lambda function.  
+
+>**NOTE**: The flow assumes you have already used Kyma console to add a service instance of an external Application to your Namespace and created a lambda with an Event trigger. 
+
+![Sending Events](./assets/knative-event-mesh-send-events.svg)
+
+
+1. The Application sends Events to [HTTP Source adapter](https://github.com/kyma-project/kyma/tree/master/components/event-sources/adapter/http) which is an HTTP server deployed inside the `kyma-integration` Namespace.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
1. The Application sends Events to the [HTTP Source adapter](https://github.com/kyma-project/kyma/tree/master/components/event-sources/adapter/http) which is an HTTP server deployed inside the `kyma-integration` Namespace.  
```

Comment (kazydek) [reply docs/event-bus/03-04-knative-eventing-mesh.md]: and not `HTTP Source Adapter` or `HTTP source adapter`?

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+### Send Events
+
+The diagram shows you the main stages of the Event flow from the moment it is sent by the external Application up to when it is received by the lambda function.  
+
+>**NOTE**: The flow assumes you have already used Kyma console to add a service instance of an external Application to your Namespace and created a lambda with an Event trigger. 
+
+![Sending Events](./assets/knative-event-mesh-send-events.svg)
+
+
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]: additional empty line

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+1. The Application sends Events to [HTTP Source adapter](https://github.com/kyma-project/kyma/tree/master/components/event-sources/adapter/http) which is an HTTP server deployed inside the `kyma-integration` Namespace.  
+
+2. The HTTP Source adapter forwards the Events to the default [Knative Broker](https://knative.dev/docs/eventing/broker-trigger).
+
+3. The Knative Broker then delivers Events to the proper lambda function. 
+
+
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]: additional empty line

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+### Subscribe to Events 
+
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]: user Namespace?

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+
+### Subscribe to Events 
+
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also use expressions that allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
You can also use expressions that allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section in [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also use expressions that allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
+
+## Test Knative Eventing Mesh
+
+To reach the new Eventing Mesh use an HTTP request with the `/events` path.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
To reach the new Eventing Mesh, use an HTTP request with the `/events` path. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+In the new Knative Eventing Mesh, you can use Knative Triggers to subscribe to any Events delivered to the Broker located in the user Namespace.  
+
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also use expressions that allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
+
+## Test Knative Eventing Mesh
+
+To reach the new Eventing Mesh use an HTTP request with the `/events` path. 
+For example, if so far you have used `gateway.example.cx/v1/events`, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
For example, if you have used `gateway.example.cx/v1/events` so far, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh. 
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+1. The Application sends Events to [HTTP Source adapter](https://github.com/kyma-project/kyma/tree/master/components/event-sources/adapter/http) which is an HTTP server deployed inside the `kyma-integration` Namespace.  
+
+2. The HTTP Source adapter forwards the Events to the default [Knative Broker](https://knative.dev/docs/eventing/broker-trigger).
+
+3. The Knative Broker then delivers Events to the proper lambda function. 
+
+
+### Subscribe to Events 
+
+
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]: additional empty line

File: docs/event-bus/03-04-knative-eventing-mesh.md (hunk 17 above)
Comment (kazydek) [reply docs/event-bus/03-04-knative-eventing-mesh.md]: or `of this document`

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+![Subscribe to Events](./assets/knative-event-mesh-subscription.svg)
+
+You can also use expressions that allow the Trigger to filter the incoming Events. For details on setting filters, read the **Trigger filtering** section [this](https://knative.dev/docs/eventing/broker-trigger/) document. 
+
+## Test Knative Eventing Mesh
+
+To reach the new Eventing Mesh use an HTTP request with the `/events` path. 
+For example, if so far you have used `gateway.example.cx/v1/events`, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh. 
+
+>**NOTE:** The HTTP Source adapter accepts only Events compliant with the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
>**NOTE:** The HTTP Source adapter only accepts Events compliant with the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 22]
```diff
@@ -0,0 +1,44 @@
+---
+title: Knative "Eventing Mesh (Alpha)"
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
title: "Knative Eventing Mesh (Alpha)"
```

File: docs/event-bus/03-04-knative-eventing-mesh.md [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+type: Details
+---
+
+## Overview
+
+Knative Eventing Mesh leverages Knative Eventing components to build an eventing mesh that provides event routing and pub/sub capabilities. It abstracts the underlying messaging system and allows you to configure different persistence for each Namespace. Kyma components wire the mesh dynamically for event routing. This way, senders can inject Events into the mesh from multiple source points, and subscribers can receive Events based on filters and their access permissions. The [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) CRDs allow the process of Event publishing and consumption to run smoother, thus significantly improving the overall performance.   
+
+ >**NOTE:** Knative Eventing Mesh is available in the alpha version. Use it only for testing purposes.
+ 
+ The new Eventing Mesh runs in parallel with the existing Event Bus. The Kyma Event Bus still supports sending Events to the regular eventing endpoint, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh.
```
Comment (kazydek) [inline docs/event-bus/03-04-knative-eventing-mesh.md]:
```suggestion
The new Eventing Mesh runs in parallel with the existing Event Bus. The Kyma Event Bus still supports sending Events to the regular eventing endpoint, while a separate Kyma endpoint handles sending Events to the new Knative Eventing Mesh. 
```

**Pull request merged**